contrib/kafka/pkg/adapter/adapter.go

```diff
@@ -68,66 +70,77 @@ func (a *Adapter) initClient() error {
 	}
 	return nil
 }
+func (a *Adapter) Cleanup(_ sarama.ConsumerGroupSession) error { return nil }
+func (a *Adapter) ConsumeClaim(sess sarama.ConsumerGroupSession, claim sarama.ConsumerGroupClaim) error {
+
+	logger := logging.FromContext(context.TODO())
+
+	for msg := range claim.Messages() {
+		logger.Info("Message topic:%q partition:%d offset:%d\n", msg.Topic, msg.Partition, msg.Offset)
```

Comment:
Think you need a `fmt.Sprintf` call or something equivalent here. The line below is the message I'm seeing.
```
"Message topic:%q partition:%d offset:%d\ndubee0 348"
```

Reply:
@dubee args - yeah... I did not mean to change yours :-)   put it in back, the origianl logger